psutil/_psutil_windows.c

```diff
@@ -2720,19 +2724,30 @@ psutil_proc_memory_maps(PyObject *self, PyObject *args) {
 
     while (VirtualQueryEx(hProcess, baseAddress, &basicInfo,
                           sizeof(MEMORY_BASIC_INFORMATION)))
+
     {
         py_tuple = NULL;
         if (baseAddress > maxAddr)
             break;
         if (GetMappedFileNameA(hProcess, baseAddress, mappedFileName,
                                sizeof(mappedFileName)))
         {
-            py_tuple = Py_BuildValue(
-                "(kssI)",
-                (unsigned long)baseAddress,
-                get_region_protection_string(basicInfo.Protect),
-                mappedFileName,
-                basicInfo.RegionSize);
+            #ifdef _WIN64
+               py_tuple = Py_BuildValue(
+                  "(KssI)",
+                  (unsigned long long)baseAddress,
```

Comment:
here you can use a single `ifdef` to enclose only the `(unsigned long / long)` logic: there's no need to re-define the whole `Py_BuildValue` block twice. Also please indent the ifdefs (no spaces)